src/unit_tests/comments_counter.test.js

```diff
@@ -0,0 +1,25 @@
+import InvolvementAPI from '../modules/involvement_api.js';
+
+const mockData = [
+  {
+    comment: 'This is nice!',
+    creation_date: '2021-01-10',
+    username: 'John',
+  },
+  {
+    comment: 'Great content!',
+    creation_date: '2021-02-10',
+    username: 'Jane',
+  },
+];
+global.fetch = jest.fn(() => Promise.resolve({
+  json: () => Promise.resolve(mockData),
+}));
+
+describe('Get comments the the length', () => {
+  test('2 Comment added count should be 2', async () => {
+    await InvolvementAPI.getComments('test');
+
+    expect(InvolvementAPI.commentCount()).toBe(2);
+  });
+});
```

Comment:
- [ ] This test is not passing:

![image](https://user-images.githubusercontent.com/57421823/157639733-e68121f5-4fc4-434a-852e-c2f9e06cfe9a.png)

Kindly correct this one.